cdap-spark-core3_2.12/src/k8s/entrypoint.sh

```diff
@@ -87,6 +90,7 @@ case "$1" in
       # launch SparkSubmit with the CDAP launcher, which will
       # setup CDAP classloading and class rewrite
       ${JAVA_HOME}/bin/java
+      ${DRIVER_EXTRA_OPTS}
```

Comment:
Is this only a problem for driver and not executor ?

Reply:
yeah, Spark does something odd with the executor where it parses the extra args into individual environment variables (https://github.com/cdapio/cdap/blob/593d0d63ec1992edc49e9bdb80d97ef9dd2ef4a9/cdap-spark-core3_2.12/src/k8s/entrypoint.sh#L41). Have verified that part is working properly.